src/Transaction.js

```diff
@@ -333,17 +345,35 @@ class Transaction {
   _increaseGasPrice() {
     const maxGasPrice = parseUnits(this.config.MAX_GAS_PRICE.toString(), 'gwei')
     const minGweiBump = parseUnits(this.config.MIN_GWEI_BUMP.toString(), 'gwei')
-    const oldGasPrice = BigNumber.from(this.tx.gasPrice)
-    if (oldGasPrice.gte(maxGasPrice)) {
-      console.log('Already at max gas price, not bumping')
-      return false
+
+    if (this.tx.gasPrice) {
+      const oldGasPrice = BigNumber.from(this.tx.gasPrice)
+      if (oldGasPrice.gte(maxGasPrice)) {
+        console.log('Already at max gas price, not bumping')
+        return false
+      }
+
+      const newGasPrice = max(
+        oldGasPrice.mul(100 + this.config.GAS_BUMP_PERCENTAGE).div(100),
+        oldGasPrice.add(minGweiBump),
+      )
+      this.tx.gasPrice = min(newGasPrice, maxGasPrice).toHexString()
+      console.log(`Increasing gas price to ${formatUnits(this.tx.gasPrice, 'gwei')} gwei`)
+    } else {
+      const oldMaxFeePerGas = BigNumber.from(this.tx.maxFeePerGas)
+      const oldMaxPriorityFeePerGas = BigNumber.from(this.tx.maxFeePerGas)
+      if (oldMaxFeePerGas.gte(maxGasPrice)) {
+        console.log('Already at max fee per gas, not bumping')
+        return false
+      }
+
+      const newMaxFeePerGas = oldMaxFeePerGas.add(minGweiBump)
+      this.tx.maxFeePerGas = min(newMaxFeePerGas, maxGasPrice).toHexString()
+      this.tx.maxPriorityFeePerGas = oldMaxPriorityFeePerGas.add(minGweiBump).toHexString()
```

Comment:
Probably will never happen in practice, but make sure that oldMaxPriorityFeePerGas is less or equal to maxFeePerGas